src/services/rhsm/rhsmServices.js

```diff
@@ -2220,25 +2164,19 @@ const getGraphCapacity = (id, params = {}, options = {}) => {
     cancel = true,
     cancelId,
     schema = [rhsmSchemas.capacity, rhsmSchemas.errors],
```

Comment:
required:
need to review this schema, we probably need to flip over to `rhsmSchemas.tally`